styles/global.css

```diff
@@ -558,3 +558,10 @@ tr:hover {
 .modal-overlay {
   background: #0a3050;
 }
+
+/* little blue pill around topics in details view */
+.pill {
+    border-radius: 2em;
+    padding: 2px 10px;
+    background-color: lightblue;
```

Comment:
add a margin-bottom of 5px or so to have som spacing between the lines

Reply:
Needed to add some more CSS, but now it works.